Core/include/Acts/Geometry/NavigationLayer.hpp

```diff
@@ -44,12 +44,12 @@ class NavigationLayer : public Layer {
   /// The binning position method
   ///
   /// @param gctx The current geometry context object, e.g. alignment
-  /// @param bValue is the value for which the binning position is requested
+  /// @param aDir is the axis direction for which the binning position is requested
```

Comment:
```suggestion
  /// @param aDir is the axis direction for which the reference position is requested
```

Reply:
There is a big chance we can get rid of the `referencePosition` and `refeferecePositionValue` eventually, as they are used in the Gen1 style geometry creation to create the BinnedArrays.

Reply:
I've fixed it though.